packages/grafana-ui/src/components/Collapse/Collapse.tsx

```diff
@@ -59,16 +60,15 @@ const getStyles = (theme: GrafanaTheme2) => ({
   `,
   header: css`
     label: collapse__header;
-    padding: ${theme.spacing(1, 2)};
+    padding: ${theme.spacing(1, 2, 0.5, 2)};
     display: flex;
     cursor: inherit;
     transition: all 0.1s linear;
     cursor: pointer;
   `,
   headerCollapsed: css`
     label: collapse__header--collapsed;
-    cursor: pointer;
-    padding: ${theme.spacing(1, 2)};
+    padding: ${theme.spacing(1, 2, 0.5, 2)};
```

Comment:
`headerCollapsed` seems to mean you don't show the collapsible icon. Seems like at that case you don't wan't to indicate clickability with `cursor: pointer`?